### ISSUE TITLE
Make it easier for user to edit issue status and type mapping

### ISSUE THREAD

**Issue by hezyin** [merico-dev/lake#399]
## Description

Right now, users need to edit a complicated string to change the configuration for issue status and type mapping. Below is an example for issue status mapping:

```
Resolved:Approved,Verified,Done,Closed;Reject:ByDesign,Irreproducible
```

Another example for issue type mapping:

```
Requirement:Story;Bug:Bug;Incident:Incident
```

## Describe the solution you'd like

The challenge is that for each issue type that users have in their Jira system, we'll allow them to configure a status mapping. 

There're two intuitive solutions that come to mind:

1. Provide a table-like input field. Users can add rows to the table. Each row has three columns: user issue type, user status type, and corresponding dev lake standard type that it maps to. 
2. Provide a button for users to add new user issue types. For each new user issue type added, we have two input fields, one for each standard type (`Resolved` and `Rejected`).

## Has the Feature been Requested Before?

No

**Comment by oddscenes** [comment]
@hezyin Made a new design idea for this approach, with tags:

**Notes (and corresponding .env variables)**
![jira-mappings-notes](https://user-images.githubusercontent.com/3789273/134080012-85d26b99-409a-4369-bfb2-1c713c11b627.png)

**No notes (same thing just clean view)**
![jira-mappings](https://user-images.githubusercontent.com/3789273/134080050-a6e5075d-ca29-46f2-b8ff-5d67a977d98d.png)

Does this cover all the type/status/custom status things?

**Comment by klesh** [comment]
closed by #517
